### ISSUE TITLE
OMV dialogs not showing titles in safari (MacOS)

### ISSUE THREAD

**Issue by jata1** [openmediavault/openmediavault#467]
This issue impacts both OMV4 and OMV5.

Looks like the issue is specific to safari. the titles render fine in chrome on my Mac

See example below but this issue impacts all edit dialog windows.

I have cleared cache and can reproduce the issue on 3 different Macs running different versions of macOS.

![Screen Shot 2019-10-16 at 11 47 10 am](https://user-images.githubusercontent.com/23280705/66984241-0eafa280-f106-11e9-9688-a925faded6cf.png)

**Comment by votdev** [comment]
Any help from someone with access to a Safari browser is welcome.

**Comment by nicolasdev** [comment]
Hi, Issue seems to be related to `theme-triton-all_1.css`, with this line commented, titles are visible. It works on Chrome to with commented line.


```
.x-component, .x-container {
    /* position:relative */
}
```

![image](https://user-images.githubusercontent.com/15873315/69097826-6737cf80-0a25-11ea-818b-cbdf4996e621.png)

**Comment by votdev** [comment]
> ```
> .x-component, .x-container {
>     /* position:relative */
> }
> ```

That looks a little bit too generic to me, but a good starting point for more investigations.

**Comment by votdev** [comment]
Does this work?
```
.x-component.x-fieldset-header-text {
	position: unset;
}
```

**Comment by nicolasdev** [comment]
Yes, it works on Safari, Chrome and Firefox (override `position: relative`)

![image](https://user-images.githubusercontent.com/15873315/69184602-2d2d0300-0ae3-11ea-914d-b8ebd9ace907.png)

**Comment by votdev** [comment]
Fixed in openmediavault 4.1.28 and 5.1.0.
